liquidity-module/index.md

```diff
@@ -359,10 +420,14 @@ Make sure to replace `cosmosxxxxx` with your address.
 You can also withdraw tokens from the pool you have created.
 
 ```bash
-gaiad tx liquidity withdraw 1 100pool-id --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443
+gaiad tx liquidity withdraw 1 100pool-id --from username --chain-id cosmoshub-testnet --gas-prices "0.025uphoton" --node https://rpc.testnet.cosmos.network:443
 ```
 
-- Replace `pool-id` with your pool id that you can see on [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx)
+Make sure to replace the first `1` with your `pool id` obtained in previous step.
```

Comment:
```suggestion
Make sure to replace the `1` with your pool id.
```

Reply:
oh, we don't have a second one in this code example, do we mean the `1`? or is `100pool-id` already provided?

Reply:
I din get the question?